### DISCUSSION TITLE
How to do a Cron execution in datahub

### DISCUSSION THREAD

**Discussion by rolandozigre** [pimcore/pimcore#16964] [Data-Hub]
Hello everyone!
I tried to do a cron execution in data but it doesn't work. I've set an execution time and run the command **datahub:data-importer:execute-cron**  but it didn't work.
I need help.

![Capture d’écran du 2024-04-18 15-19-14](https://github.com/pimcore/pimcore/assets/160744678/758f00a8-0cf7-4381-9d3b-9b580a52a3e0)

**Comment by skoch98** [comment]
Hi @rolandozigre 

for me in the newest pimcore & data hub & data importer version it works fine.

1) I created a new data importer configuration
2) I ensured that the new configuration is marked as active
3) I definied the cron syntax to `58 * * * *`
4) I executed `php bin/console datahub:data-importer:execute-cron` on cli and the job was planned:
![image](https://github.com/pimcore/pimcore/assets/31919154/309ecdce-0782-4d2f-96c2-e75c598bda99)
5) I executed `php bin/console datahub:data-importer:process-queue-sequential ` for "Sequentiell" setting under "Import settings -> settings". For  "Sequentiell" use  `php bin/console datahub:data-importer:process-queue-parallel --processes=5`. And the data was imported successfully:
![image](https://github.com/pimcore/pimcore/assets/31919154/4cad4173-07e7-4363-a033-84c36b4d3fd8)

Please make sure that you config is valid and activated. If this is already the case please run `php bin/console datahub:data-importer:execute-cron` manually and share the output with us.
Also make sure that the import data are provided.

**Reply by rolandozigre** [reply]
Hi **@skoch98**,
Ok, I'll try it and get back to you
